mmdet/core/bbox/iou_calculators/iou2d_calculator.py

```diff
@@ -137,7 +137,7 @@ def bbox_overlaps(bboxes1, bboxes2, mode='iou', is_aligned=False, eps=1e-6):
             When using FP16, we can reduce:
                 R = 11 x N * 4 / 2 Byte
 
-        So do the 'giou' (large than 'iou').
+        So do the 'giou' or 'diou' (large than 'iou').
```

Comment:
need to add unit test

Reply:
Where is an example of unit test such as 'goiu'?

Reply:
you need to test the diou.